moto/core/models.py

```diff
@@ -788,6 +789,27 @@ def __call__(self, func=None):
         else:
             return mocked_backend
 
+ecs_backends_global = {}
+class atidot_base_decorator(base_decorator):
+    mock_backend = MockAWS
+
+    def __init__(self, backends):
+        super().__init__(backends)
```

Comment:
do this here -
~~~python
     for backend in self.backends:
            self.backends[backend].at_callback = callback
            # why not?
            # backend.at_callback = callback
~~~

Reply:
backend is a name of region...(str type)